package.json

```diff
@@ -3,40 +3,45 @@
   "version": "23.2.6",
   "license": "MIT",
   "description": "actual syncing server",
-  "main": "index.js",
+  "type": "module",
   "scripts": {
     "start": "node app",
     "lint": "eslint .",
     "build": "tsc",
-    "test": "jest",
+    "test": "NODE_OPTIONS='--experimental-vm-modules --trace-warnings' jest",
     "types": "tsc --noEmit --incremental",
     "verify": "yarn -s lint && yarn types"
   },
   "dependencies": {
     "@actual-app/api": "4.1.6",
     "@actual-app/web": "23.2.5",
     "bcrypt": "^5.1.0",
-    "better-sqlite3": "^7.5.0",
+    "better-sqlite3": "^8.0.1",
```

Comment:
I'm worried if this will not create some problems as now we will have 2x different `better-sqlite3` versions here. Even if you are confident with this change - would you mind doing it in a separate PR so it's easier to roll back if-need-be?


```
$ yarn why better-sqlite3
├─ @actual-app/api@npm:4.1.6
│  └─ better-sqlite3@npm:7.5.0 (via npm:^7.5.0)
│
└─ actual-sync@workspace:.
   └─ better-sqlite3@npm:8.1.0 (via npm:^8.0.1)
```